spectral_cube/io/casa_image.py

```diff
@@ -57,14 +58,14 @@ def load_casa_image(filename, skipdata=False, memmap=True,
 
     # read in the data
     if not skipdata:
-        data = image_to_dask(filename, memmap=memmap)
+        data = image_to_dask(filename, memmap=memmap, target_chunksize=target_chunksize)
 
     # CASA stores validity of data as a mask
     if skipvalid:
         valid = None
     else:
         try:
-            valid = image_to_dask(filename, memmap=memmap, mask=True)
+            valid = image_to_dask(filename, memmap=memmap, mask=True, target_chunksize=target_chunksize)
```

Comment:
Noting for the future: I think eventually we should expand this to allow a memory limit (e.g., 8 * u.Mb) or exact array chunk shape.